definitions/npm/multimatch_v2.x.x/flow_v0.37.x-/multimatch_v2.x.x.js

```diff
@@ -0,0 +1,23 @@
+declare module "multimatch" {
+  declare type Options = {
+    debug: boolean,
+    dot: boolean,
+    flipNegate: boolean,
+    matchBase: boolean,
+    nobrace: boolean,
+    nocase: boolean,
+    nocommnet: boolean,
+    noext: boolean,
+    noglobstar: boolean,
+    nonegate: boolean,
+    nonull: boolean
+  };
```

Comment:
When you define options, do you really have to define all of the properties? If not, i'd define the object properties as optional: `nonull?: boolean`. I also think that it would make sense to make the Options exact type(`{|prop: boolan, ...etc|}`: It probably doesn't make any sense to pass unknown props.

Reply:
:+1: makes sense, that's exactly how the options object is treated. Will make amends :raised_hands:!